README.md

```diff
@@ -578,6 +578,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
 - ### **R**
 
+  - [Rayan Abid](https://github.com/RayanAbid)
```

Comment:
Please add your name in line 602, after Raveena Bhasin.

Reply:
Sure will do that. Will I have to create a new commit for that?